constitution/constitution.tex

```diff
@@ -32,13 +32,19 @@ \section{}
 \article{Officers \& Advisor}
 
 \section{}
-Section 1: The Officers of the Organization shall be President, Vice President, Treasurer, and Secretary.
+The Officers of the Organization shall be President, Vice President, Treasurer, Secretary, Social Media, and Lab Manager.
 
 \section{}
-The Officers of the Organization shall be elected by a simple majority of the members attending the election meeting who have served on an SCR team.
+The Voting Membership shall be defined as any member who is an active Officer, a Team Captain, or a Team Lead. SCR members may request to the Officers to become Voting Membership until the end of the academic year.
 
 \section{}
-Any Officer or Advisor may be removed by a three-fourths vote of the qualified voting membership. 
+The Officers of the Organization shall be elected by the Voting Membership.
+
+\section{}
+Any Officer or Advisor may be removed by a two-thirds votes of the Officers.
+
+\section{}
+Any Officer or Advisor may be removed by a three-fourths vote of those in attendance at an Officer Impeachment meeting. A petition with at least one-third of the Voting Membership shall result in an Officer Impeachment meeting within two weeks of when the petition is brought to the President or Vice President. The meeting time shall be voted on by the Voting Membership and the Officers.
 
 \section{}
 The President shall be the chief executive officer of the Organization.  The President shall preside at all meetings and direct the affairs of the Organization with the advice and consent of the other officers.  The President shall decide what competition teams may be a part of the Organization based on the Organization’s current capabilities. The President shall ensure that the goals of leadership and success are met by all. In the event that there is no current President, the remainder of the President’s term shall be filled in the following order of succession:  Vice-President, then Treasurer, then Secretary, then a qualified student of the Advisor’s choice.
```

Comment:
I still hold that Secretary should be ranked above Treasurer, here and in all other places officers are listed or put in hierarchy.